transport/http/client.go

```diff
@@ -75,11 +74,11 @@ func (c Client) Endpoint() endpoint.Endpoint {
 
 		req, err := http.NewRequest(c.method, c.tgt.String(), nil)
 		if err != nil {
-			return nil, fmt.Errorf("NewRequest: %v", err)
+			return nil, Err{"NewRequest", err}
```

Comment:
Do you perfer a verbose constant, like:

```go
const (
	DomainRequestGeneration = "NewRequest"
)
```

Or would you prefer something smaller like:

```go
const (
	DomNewRequest = "NewRequest"
)
```